tests/Resources/Common.robot

```diff
@@ -243,3 +244,22 @@ Close Generic Modal If Present
         Click Element    xpath=//button[@aria-label="Close"]
         Wait Until Generic Modal Disappears
     END
+
+Extract Value From JSON Path
+    [Documentation]    Given a Python JSON Object (i.e., a dictionary) and
+    ...                a desired path (e.g., spec.resources.limits.cpu), it retrieves
+    ...                the value by looping into the dictionary.
+    [Arguments]    ${json_dict}    ${path}
+    ${path_splits}=    Split String    string=${path}    separator=.
+    ${value}=    Set Variable    ${json_dict}
+    FOR    ${idx}    ${split}    IN ENUMERATE    @{path_splits}  start=1
+        Log    ${idx} - ${split}
+        ${value}=    Set Variable    ${value["${split}"]}
+    END
```

Comment:
what happens if one of the keys is not found? I would do some error checking here

Reply:
hm yeah it would fail. I thought error handling wasn't acutally necessary since if you pass a wrong path it fails as it's right to do. 

But, a better approach could be to return something empty in case one of the keys doesn't exist. Would it be ok?

Reply:
I applied an error handling. Could you check?

Reply:
now that we upgraded to rf5 you could use try/except: https://github.com/robotframework/robotframework/blob/master/doc/releasenotes/rf-5.0.rst#tryexcept